python/pyspark/errors/tests/connect/__init__.py

```diff
@@ -0,0 +1,16 @@
+#
```

Comment:
For adding a new package, you should also add it at `setup.py` at https://github.com/apache/spark/blob/master/python/packaging/client/setup.py#L185 and https://github.com/apache/spark/blob/master/python/packaging/classic/setup.py#L315

Reply:
(So they are tested with connect mode without jars)